site/src/pages/docs/server-timeouts.mdx

```diff
@@ -0,0 +1,169 @@
+# Overriding server timeouts
+
+There are several ways to override the default timeouts at server-side.
+
+## Global Configuration
```

Comment:
What do you think of moving `Global Configuration` to the bottom?
I guess flags are probably the least preferred type because it needs extra JVM properties or SPI. It's good to go from a large range to a small range, but I think it's okay to change the order a little to put the information you're interested in first.

Reply:
I think It is good to move commonly used methods to the top. 👍